.github/workflows/agenda.yaml

```diff
@@ -0,0 +1,53 @@
+name: Agenda Generator
+
+on: # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '15 12 * * 1' # run every Monday at 12:15 PM UTC
+  workflow_dispatch:
+    branches:
+      - main
+      - agenda_workflow
+
+permissions:
+  discussions: write
+
+jobs:
+  create-discussion:
+    name: discussion-gen
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Print discussion date
+      id: meeting-date
+      run: |
+        meet_date=$(python3 -c 'import datetime; print((datetime.datetime.now() + datetime.timedelta(days=7)).strftime("%m/%d/%Y"))')
+        echo "meetdate=$meet_date" >> $GITHUB_OUTPUT
+    - name: Create a new GitHub Discussion
+      id: create-discussion
+      uses: octokit/graphql-action@v2.x
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        query: |
+          mutation {
+            # input type: CreateDiscussionInput
+            createDiscussion(input: {
+                                      repositoryId: "${{ vars.REPO_ID }}",
+                                      categoryId: "${{ vars.CAT_ID }}",
+                                      body:
+                                        "Welcome to our community call. This call will be held on ${{ steps.meeting-date.outputs.meetdate }} at [8:30 AM EDT (UTC-4)](https://mytime.io/08:30/EDT).
+                                        Click on [Primaza Google Meet](https://meet.google.com/rha-ftwg-orr) to join us. You can also join the [Primaza Group](https://groups.google.com/g/primaza/members) to receive a recurring invite on your calendar.
+                                        Agenda Items are entered through comments. Feel free to add your agenda Items.",
+                                      title: "Community Call Agenda ${{ steps.meeting-date.outputs.meetdate }}"}) {
```

Comment:
We could add some emojis to make this post more attractive. This may be helpful: https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md

Reply:
I did check it out https://github.com/dperaza4dustbit/primaza/discussions/30